buildSrc/src/integTest/java/org/elasticsearch/gradle/precommit/ThirdPartyAuditTaskIT.java

```diff
@@ -49,18 +49,6 @@ public void testElasticsearchIgnored() {
         assertNoDeprecationWarning(result);
     }
 
-    public void testWithEmptyRules() {
```

Comment:
Why has this been removed?

Reply:
Forbidden api behavior has changed here. It no longer errors on empty rule files. Only if file itself is missing or not specified. I don't think it's worth adding additional checks on our end to reinstate that behavior.